src/service.js

```diff
@@ -1,166 +1,172 @@
-import LambdaRestClient from '@lambdatest/node-rest-client'
-import logger from '@wdio/logger'
+"use strict";
 
-const log = logger('@wdio/lambdatest-service')
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+exports.default = void 0;
 
-export default class LambdaRestService {
-    constructor() {
-        this.testCnt = 0
-        this.failures = 0 // counts failures between reloads
+require("source-map-support/register");
+
+var _nodeRestClient = _interopRequireDefault(require("@lambdatest/node-rest-client"));
+
+var _logger = _interopRequireDefault(require("@wdio/logger"));
+
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+
+const log = (0, _logger.default)('@wdio/lambdatest-service');
+
```

Comment:
These lines should use const instead of var.
We should not use _interopRequireDefault. lets look for alternatives